deployment/helm/resource-management-policies/balloons/templates/configmap_enable_nri.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.nri.enable }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: patch-containerd-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "balloons-plugin.labels" . | nindent 4 }}
+data:
+  patch_nri.sh: |
+   #!/bin/bash
+
+   config_file="/home/config.toml"
+   plugin="[plugins.\"io.containerd.nri.v1.nri\"]"
+   switch="disable = false"
+   
+   cp /etc/containerd/config.toml "$config_file"
+
+   # Check if the plugin exists in the config file
+   if grep -q "$plugin" "$config_file"; then
+     # Check if the nri plugin is already enabled
+     if grep -q "disable = false" "$config_file"; then
+       echo "NRI is already enabled..."
+     else
+       # Enable the NRI plugin
+       sed -i "s/disable = true/$switch/" "$config_file"
+       echo "NRI has been enabled..."
+       cp -f "$config_file" /etc/containerd/config.toml
+       echo "Restarting containerd...."
+       nsenter -t 1 -m bash
```

Comment:
do you really need bash here or just execute systemctl in ns?

Reply:
only systemctl isn't enough, I also need something to read the file and modify it is necessary